node/mock-test.ts

```diff
@@ -1,19 +1,25 @@
 import cp = require('child_process');
 import fs = require('fs');
-import path = require('path');
+import ncp = require('child_process');
 import os = require('os');
+import path = require('path');
 import cmdm = require('./taskcommand');
+import semver = require('semver');
 import shelljs = require('shelljs');
+import syncRequest = require('sync-request');
 
 const COMMAND_TAG = '[command]';
 const COMMAND_LENGTH = COMMAND_TAG.length;
+const testDirectory = path.join(__dirname, '_test');
```

Comment:
as discussed let's create in home dir, appdata etc...  Not in a folder that gets blown away every time